translations/README.es.md

```diff
@@ -0,0 +1,56 @@
+![Guía freeCodeCamp](https://s3.amazonaws.com/freecodecamp/wide-social-banner.png)
+
+# Guía freeCodeCamp 
+
+[![Build Status](https://img.shields.io/travis/freeCodeCamp/guides/master.svg?style=flat-square)](https://travis-ci.org/freeCodeCamp/guides) [![license](https://img.shields.io/badge/license-BSD--3--Clause-lightgrey.svg?style=flat-square)](https://opensource.org/licenses/BSD-3-Clause)  [![Gitter](https://img.shields.io/gitter/room/freeCodeCamp/Contributors.svg?style=flat-square)](https://gitter.im/freeCodeCamp/Contributors)
+
+[![GitHub Issues](https://img.shields.io/github/issues/freeCodeCamp/guides.svg?style=flat-square)](https://github.com/freeCodeCamp/guides/issues) [![GitHub Pull Requests](https://img.shields.io/github/issues-pr/freeCodeCamp/guides.svg?style=flat-square)](https://github.com/freeCodeCamp/guides/pulls) [![Pull Requests Bienvenida](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
+[![Novatos Amigable](https://img.shields.io/badge/first--timers--only-friendly-blue.svg?style=flat-square)](http://www.firsttimersonly.com/)
+
+[![Throughput Graph](https://graphs.waffle.io/freeCodeCamp/guides/throughput.svg)](https://waffle.io/freeCodeCamp/guides/metrics)
+
+La comunidad freeCodeCamp esta creando una **"Guia"** masiva.
+
+Esta herramienta de referencia eventualmente incluira miles de articulos que cubriran todas las areas de desarrollo, diseño y ciencia de datos - todo escrito para ser facilmente comprendido por personas nuevas en la programación.
```

Comment:
Wouldn't it be better as Data science instead of ciencia de datos?
The first althought is in English is widely used in Spanish speaking countries including by mainstream media.

Reply:
yes, thanks for the comment! i was not really sure which IT terms should be kept in spanish and which ones in english.